lib/clean.js

```diff
@@ -1,19 +1,15 @@
 'use strict';
 
-var merge = require('lodash').merge;
-var del = require('del');
+import {merge} from 'lodash';
+import del from 'del';
 
-var defaults = {
+const defaults = {
   dest: 'dist'
 };
 
-module.exports = function (gulp, options) {
+module.exports = (gulp, options) => {
   var opts = merge({}, defaults, options);
   var dest = opts.dest;
 
-  gulp.task('clean', function (done) {
-    del(dest).then(function () {
-      done();
-    });
-  });
+  gulp.task('clean', done => del(dest).then(() => done()));
```

Comment:
I find this formatting challenging. I need to do some reading on formatting for multi-line anonymous callbacks. My first swag would be something more like:

```
gulp.task('clean', done => {
  del(dest)
    .then(done);
});
```

Note I also think you should be able to pass the `done` function reference without wrapping it in a callback and invoking it :).